core/src/main/java/greencity/config/SecurityConfig.java

```diff
@@ -84,7 +83,13 @@ protected void configure(HttpSecurity http) throws Exception {
             .accessDeniedHandler((req, resp, exc) -> resp.sendError(SC_FORBIDDEN, "You don't have authorities."))
             .and()
             .authorizeRequests()
-            .antMatchers("/management/**", "/css/**", "/img/**").hasRole(ADMIN)
+            .antMatchers(
+                "/management/**",
+                "/css/**",
+                "/img/**",
```

Comment:
Static files must be accessible for all

Reply:
I didn’t change it. Those files already have been there.